test/integration/singlecluster/controller/jobs/appwrapper/appwrapper_controller_test.go

```diff
@@ -830,7 +830,7 @@ var _ = ginkgo.Describe("AppWrapper controller interacting with scheduler", gink
 		gomega.Expect(createdAppWrapper.Spec.Components[0].PodSetInfos[0].NodeSelector[instanceKey]).Should(gomega.Equal(spotUntaintedFlavor.Name))
 		gomega.Expect(createdAppWrapper.Spec.Components[1].PodSetInfos[0].NodeSelector[instanceKey]).Should(gomega.Equal(onDemandFlavor.Name))
 		util.ExpectPendingWorkloadsMetric(clusterQueue, 0, 0)
-		util.ExpectReservingActiveWorkloadsMetric(clusterQueue, 1)
+		util.ExpectAdmittedWorkloadsTotalMetric(clusterQueue, "", 1)
```

Comment:
Does it eliminate all asserts on `ExpectReservingActiveWorkloadsMetric`? I think it would be good to keep one or two just as sanity checks for the function. However, we would need to assert on them after which assert on ExpectAdmittedWorkloadsTotalMetric. So basically keep:

```
util.ExpectAdmittedWorkloadsTotalMetric(clusterQueue, "", 1)
// assert also for the reserving_active_workloads metric as a sanity check for
// correctness of the metric. We do this assert after verifying ExpectAdmittedWorkloadsTotalMetric
// to make sure the update request is succeeded. 
util.ExpectReservingActiveWorkloadsMetric(clusterQueue, 1)
```
Feel free to rephrase, but I would like to keep still one or two dedicated asserts.